### DISCUSSION TITLE
Model’s column alias question (accessors/old_values/new_values)

### DISCUSSION THREAD

**Discussion by l2aelba** [owen-it/laravel-auditing#965] [Q&A] [answered]
I followed this tut: https://medium.com/@steno983/laravel-models-column-alias-c48a75065fb9

The main point is changing the column name in the database from `customer_name` to `name` to align with Laravel's model field.

**Example:**

```php
class Customer extends Model implements AuditableContract
{
  use Auditable;
  protected $table = 'customer';


  // Accessors
  protected function name(): Attribute
  {
    return Attribute::make(
      get: fn(mixed $value, array $attributes) => $attributes['customer_name'],
      set: fn(string $value) => [
        'customer_name' => $value,
      ],
    );
  }

  // Append
  protected $appends = ['name'];

  // Hide
  protected $hidden = ['customer_name'];
}
```

The `old_values` / `new_values `are still showing like this::

```
{ "customer_name": "…" }
```

not as expected result that I want like this…:

```
{ "name": "…" }
```

Is there anything I'm missing, or is there a way I can improve this?

**Comment by parallels999** [comment, marked as answer]
https://github.com/owen-it/laravel-auditing/blob/9357577514120015961e1ae90f0ee3b6d6193de6/config/audit.php#L71-L75
https://github.com/owen-it/laravel-auditing/blob/9357577514120015961e1ae90f0ee3b6d6193de6/src/Auditable.php#L99
https://github.com/owen-it/laravel-auditing/blob/9357577514120015961e1ae90f0ee3b6d6193de6/src/Auditable.php#L125-L126
It is the expected behaivor

**Reply by parallels999** [reply]
There is an open issue, #828
IMO, these attributes should not be audited, it could be OPT-IN; 
Nor has anyone come forward who is committed to making that change and maintaining it.

**Reply by l2aelba** [reply]
Thanks for the info, then I will wait for it :)
